tests/FixturesLoader.php

```diff
@@ -102,6 +102,9 @@ protected function loadImportFiles(array $files)
         //         "/project/features/Context/catalog/footwear/jobs.yml"
         //         "/project/features/PimEnterprise/Behat/Context/../../../Context/catalog/footwear/jobs.yml"
         // ]
+        if (empty($files)) {
```

Comment:
Actually you shouldn't do that because even if we don't have any file to import in our test catalog, we still need to import the minimal catalog from the InstallerBundle.